### PR TITLE
mature interface of command line

### PR DESCRIPTION
**Pull request by olebedev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,13 +7,14 @@
...
     "express": "^4.6.1",
     "express-react-views": "^0.5.0",
-    "react": "~0.11",
     "node-jsx": "^0.11.0",
+    "react": "~0.11",
     "react-tools": "~0.11",
-    "ws": "^0.4.31",
-    "swarm": "git+https://github.com/gritzko/swarm.git"
+    "swarm": "git+https://github.com/gritzko/swarm.git",
+    "ws": "^0.4.31"
```
Comment (gritzko) [inline package.json:17]: why the reordering?

**Comment by gritzko** [comment]
Oleg, some remarks on the commit:
- When doing minor changes, please don't reformat the rest of the file; that is really confusing. If it needs some refactoring or reindenting, please do that in a separate commit.
- I spent some time trying to understand why did you reorder deps.
- The key piece of logic became an anonymous function at the 5th level of indentation (I mean the recursive call + sending the response).
  Long story short, please remember that somebody is actually reading it all.

**Comment by gritzko** [comment]
P.S. 20f2e1e39e8bdcb2a922234ea7b3a49a042a2968

File: package.json (hunk 1 above)
Comment (olebedev) [inline package.json:17]: When you install deps by `npm i some --save`, `npm` save deps into `package.json` file and reorder keys by self.

**Comment by olebedev** [comment]
- of course, I did it because of the need to move the the application start logic to the function
- see [this](https://github.com/gritzko/todomvc-swarm/pull/12#discussion_r18440018)
- I don't find what you talking about

So, maybe I should create an issue in the next time? Before implementing.
